src/oic/utils/sdb.py

```diff
@@ -1018,44 +1077,50 @@ def get_client_id_for_session(self, sid):
 
     def get_client_ids_for_uid(self, uid: str) -> List[str]:
         """Return client_ids for a given uid."""
-        return self._db.get_client_ids_for_uid(uid)
```

Comment:
These changes seem unnecessary. They were already implemented using the SessionDB API so why reimplement them differently?

Reply:
I would argue that the changes make the implementation more robust.
Handles a possible exception that the original doesn't.

Reply:
Raising `KeyError` for missing `uid` seems fine for me and we should always have `client_id` in session. So the benefit doesn't look that great to me.

But if you really want to do this, than it should be done in `SessionBackend`.

Reply:
I guess this is outdated.

Reply:
Hm, I still see it as changed...